dev/run-tests

```diff
@@ -20,4 +20,8 @@
 FWDIR="$(cd "`dirname $0`"/..; pwd)"
 cd "$FWDIR"
 
-exec python -u ./dev/run-tests.py "$@"
+set -e
+for i in `seq 1 300`; do
```

Comment:
Try to run this multiple times and see if it won't hang in DataFrameReaderWriterSuite